Elevel/Domain/Enums/Level.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Elevel.Domain.Enums
 {
-    public enum Level
+    public enum Level : byte
```

Comment:
Why exactly byte?
it means that if we get the argument in int type we will have to convert it into byte

Reply:
it's OK, in DB for level will be used byte instead of int, it reduces memory usage. one remark: try to always specify number enums with exact number to avoid pain in future when you will need to add another enum value and place it in the middle